src/ui/defaulteffects.ui

```diff
@@ -51,11 +51,40 @@
          <string>None</string>
         </property>
        </item>
+
+        <!-- Bronco 40 effects -->
```

Comment:
Remark: The .ui files are created by Qt Designer (usually) , so comments wont last long here. Having the comments in the source files is enough.